projects/storefrontstyles/scss/cxbase/mixins/_visible-focus.scss

```diff
@@ -4,4 +4,15 @@
   outline-width: var(--cx-visual-focus-width, 2px);
   outline-offset: 4px;
   transition: none;
+
+  // TODO: Remove feature flag next major
```

Comment:
We should include the cleanup ticket #.

Reply:
Done: https://jira.tools.sap/browse/CXSPA-5912